stl/inc/ranges

```diff
@@ -8402,6 +8402,9 @@ namespace ranges {
     _EXPORT_STD template <class _Container, input_range _Rng, class... _Types>
         requires (!view<_Container>)
     _NODISCARD constexpr _Container to(_Rng&& _Range, _Types&&... _Args) {
+        static_assert(!is_const_v<_Container>, "C must not be const. (N4928 [range.utility.conv.to])");

```

Comment:
These are all citing N4928, but this LWG issue was resolved after that Working Paper was published. (Observe how the proposed resolution says "This wording is relative to N4928.") Therefore, these Mandates don't actually appear in N4928.

Since we don't have the next WP number yet, we can simply cite the stable name alone. The Working Paper number is only a must-have when there's a paragraph citation (as paragraphs are renumbered very frequently).